.github/ISSUE_TEMPLATE/intent-to-implement--i2i-.md

```diff
@@ -48,6 +56,15 @@ The launch tracker is meant to be an easy way of sharing a project's status with
 You should add a link to the launch tracker for this work here if applicable. One ideal template and instructions can be found here: bit.ly/amp-launch-tracker
 -->
 
+## Consideration checklist
+### Design
+- [ ] Design shows understanding of problem statement/priority
+- [ ] Acceptance/success criteria for problem being solved is clear
+- [ ] Design socialized
+<!--
+Mark as done once this design has been presented at a [Design Review](https://github.com/ampproject/amphtml/blob/532dbbbc15c879a32e26e1e890b3a6b5b71cb123/contributing/design-reviews.md).
```

Comment:
Switch to this URL: https://github.com/ampproject/amphtml/blob/master/contributing/design-reviews.md